docs/driver.md

```diff
@@ -12,14 +12,20 @@ The command used to launch the IDE. By default, the driver uses the `idea` comma
 By default, the driver waits `30 seconds` for the IDE to connect the probe. After exceeding this time, the whole test fails.  
 #### Automatic checks
 ```
-driver.checks {
-    errors = false
+driver.check {
+     errors {
+       enabled = false
+       exclude-messages = []
+       include-messages = [".*"]
```

Comment:
I added `-messages` because include is special keyword in HOCON and could not use it (means It is possible but need to wrap `include` in additional double quotes)

Reply:
Wow I did know that `include` is a special keyword but didn't realize it's forbidden even as a key 😯

Reply:
(unless escaped, yeah)

Reply:
I know that `include-messages` looks worse than simply `include`, but probably `include-messages` is better than `"include"`

Reply:
Alternatively "includes" and "excludes" or "included" and "excluded"